x-pack/plugins/endpoint/public/embeddables/resolver/view/graph_controls.tsx

```diff
@@ -85,40 +85,66 @@ export const GraphControls = styled(
         <div className={className}>
           <EuiPanel className="panning-controls" paddingSize="none" hasShadow>
             <div className="panning-controls-top">
-              <button className="north-button" title="North" onClick={handleNorth}>
+              <button
+                className="north-button"
+                data-test-subj="north-button"
+                title="North"
+                onClick={handleNorth}
+              >
                 <EuiIcon type="arrowUp" />
               </button>
             </div>
             <div className="panning-controls-middle">
-              <button className="west-button" title="West" onClick={handleWest}>
+              <button
+                className="west-button"
+                data-test-subj="west-button"
+                title="West"
+                onClick={handleWest}
+              >
                 <EuiIcon type="arrowLeft" />
               </button>
-              <button className="center-button" title="Center" onClick={handleCenterClick}>
+              <button
+                className="center-button"
+                data-test-subj="center-button"
+                title="Center"
+                onClick={handleCenterClick}
+              >
                 <EuiIcon type="bullseye" />
               </button>
-              <button className="east-button" title="East" onClick={handleEast}>
+              <button
+                className="east-button"
+                data-test-subj="east-button"
+                title="East"
+                onClick={handleEast}
+              >
                 <EuiIcon type="arrowRight" />
               </button>
             </div>
             <div className="panning-controls-bottom">
-              <button className="south-button" title="South" onClick={handleSouth}>
+              <button
+                className="south-button"
+                data-test-subj="south-button"
+                title="South"
+                onClick={handleSouth}
+              >
                 <EuiIcon type="arrowDown" />
               </button>
             </div>
           </EuiPanel>
           <EuiPanel className="zoom-controls" paddingSize="none" hasShadow>
-            <button title="Zoom In" onClick={handleZoomInClick}>
+            <button title="Zoom In" data-test-subj="zoom-in" onClick={handleZoomInClick}>
```

Comment:
the zoom-out and zoom-in are only called 5 times a piece, but it may be enough that we'd want to use a constant for them?  up to you.

Reply:
What you mean by constant